playbooks/pre-install-playbook/roles/viya-ark.preinstall/tasks/pre.required_packages_config.yml

```diff
@@ -31,7 +31,7 @@
       - "{{ ansible_os_family }}-{{ ansible_architecture }}.yml"
 
   - block:
-    ## RHEL 8 Specific package requirements
+    ## RHEL 8, and RHEL 8 compatible systems, specific package requirements
```

Comment:
```suggestion
    ## RHEL 8 and RHEL 8 compatible systems, specific package requirements
```

Reply:
If first comma comes out, then second one should as well.   They were added a parenthetical to the new wording, but the sentence is fine w/out, so removing.